src/PrestaShopBundle/Command/views/cqrs-commands-list.md.twig

```diff
@@ -0,0 +1,15 @@
+### Commands
```

Comment:
As explained above I think ONE file for ALL the commands would not be pratical to read or research, we should split them:
- either one command / one page (in a logical structure based on the domain), and each domain would have a list page (easy with ugo `{children}`)
- maybe one page per sub domain to avoid having too many (one page for Order domain, one page for Product domain, ...) But maybe at least split queries and commands into two different pages

Reply:
hm ok so, we can have a table with command & queries, which would only contain command class and respective handler, but those should be links to a dedicated page where we can show detailed information of single command.
Right? Would It be possible?